app/views/media_objects/_resource_description.html.erb

```diff
@@ -18,7 +18,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 <%= form_for @media_object, html: { class: 'form-vertical' } do |form| %>
   <%= hidden_field_tag :step, @active_step %>
 
-  <%= render partial: 'text_field',
+  <%# render partial: 'text_field',
```

Comment:
I'd get rid of it rather than commenting out -- otherwise it just provides noise for the next person reading the file.

Reply:
Too efficient... a question was coming. Will remove. Was wondering though from past experience if commenting out would leave a trail to make merging from upsteam easier by indicating explicitly communicating removal of the feature?

Reply:
I you really wanted a trail, I would recommend leaving a comment like `<# Bibliographic import from stock Avalon removed %>`. One of the issues with ERB is that commented out code looks a lot like regular code at a casual glance.

My personal preference would be to just remove it, but either way works, and if you prefer the comment I'm fine with it.